source/redux/parts/login.js

```diff
@@ -39,6 +39,13 @@ const showNetworkFailureMessage = () =>
 		[{text: 'OK'}],
 	)
 
+const showServerErrorMessage = () =>
+	Alert.alert(
+		'Server Failure',
+		'The OleCard Server is having issues right now. Please try again later.',
```

Comment:
This language is problematic.

Something more along the lines of:

```
Unable to get a good response from the OleCard server; please try again later.
```

seems like it'd be less likely to cause a student to complain to IT, for example. Puts the blame on us rather than IT, even though (after scrubbing forward in the commits) it seems we're only showing this when we get a 5xx.

Reply:
I think I'd interpret yours in the same way as mine, ie as though it's an IT problem. 

The clearest thing would be to just say something like "We're having issues talking to OleCard. Pleas try again later"?

Reply:
@hawkrives, yes, that's the idea. It's probably better to just do `We're having issues talking to OleCard. Please try again later.` Even if the issue is on their end, we're accepting responsibility that it might be because we're triggering an exception due to an incorrect request format.